src/main/java/model/RandomGenerator.java

```diff
@@ -0,0 +1,23 @@
+package model;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Random;
+
+public class RandomGenerator {
+    private static final Random random = new Random();
+
+    private static int makeRanNum() {
+        return random.nextInt(10);
+    }
+
+    public static List<Integer> makeRanNumList(int trialNum) {
+        List<Integer> randomNumList = new ArrayList<>();
+
+        for (int i=0; i<trialNum; i++) {
+            randomNumList.add(makeRanNum());
+        }
```

Comment:
코드 전체적으로 코드 자동 정렬 진행부탁드려요. 🙏
[이전에 코멘트](https://github.com/next-step/java-racingcar-kakao/pull/31#discussion_r1542361662)를 참고해서 진행주시면 될 것 같아요. 

```suggestion
        for (int i = 0; i < trialNum; i++) {
            randomNumList.add(makeRanNum());
        }
```